utils/get_supported_scylla_base_versions.py

```diff
@@ -95,8 +95,16 @@ def get_supported_scylla_base_versions(self, supported_versions) -> list:  # pyl
                     # The dest version is a minor release of latest enterprise release
                     ent_base_version.append(version)
                 else:
-                    # Choose the last two releases as upgrade base
-                    ent_base_version += ent_release_list[idx-1:][:2]
+                    lts_version = re.compile(r'\d{4}\.1')  # lts = long term support
+                    sts_version = re.compile(r'\d{4}\.2')  # sts = short term support
+
+                    if lts_version.search(version):
+                        # we need to support last version + last LTS version
+                        idx = 2 if idx == 1 else idx
+                        ent_base_version += ent_release_list[idx - 2:][:2]
+                    elif sts_version.search(version):
```

Comment:
I think the `else` clause must also be added for safety. For now we get silent `nothing`.

Reply:
Print of `warning` will not harm but provide useful info for possible debugging.

Reply:
although i think it shouldn't happen, and this is not really useful, i added a warning message